### PR TITLE
npm-install: use --no-audit and --no-fund by default

### PR DESCRIPTION
**Pull request by MylesBorins**
This can speed up install time by a non-trivial amount.

##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] contribution guidelines followed
      [here](https://github.com/nodejs/citgm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by MylesBorins** [comment]
Thanks @darcy for the suggestion

File: lib/package-manager/install.js [hunk 1]
```diff
@@ -31,6 +31,11 @@ function install(packageManager, context) {
       `Using temp directory: "${options.env['npm_config_tmp']}"`
     );
 
+    if (packageManager === 'npm') {
+      args.push('--no-audit');
```
Comment (ljharb) [inline lib/package-manager/install.js:35]: these could also be added in an .npmrc file

Comment (MylesBorins) [reply lib/package-manager/install.js:35]: We would have to automate creating a .npmrc and install it at the root of every module when we extract the tarball... seems like way more work that doing this.

Comment (ljharb) [reply lib/package-manager/install.js:35]: They’re read up the filesystem, so putting it in the root of this repo seems like it should be enough - but fair :-)

Comment (MylesBorins) [reply lib/package-manager/install.js:35]: That would not work with the architecture of CITGM. Every module is un archived into it's own file in the temp dir and `npm install` is run in that directory being orchestrated by CITGM

**Pull request merged**